packages/reassure-cli/src/commands/measure.ts

```diff
@@ -64,15 +65,16 @@ export async function run(options: MeasureOptions) {
   const testRunnerArgs = process.env.TEST_RUNNER_ARGS ?? defaultArgs;
 
   const nodeArgs = [
-    '--jitless',
+    // Use less restrictive flags to allow running WASM code (e.g. `fetch`) in tests.
+    options.enableWasm ? '--no-opt --no-sparkplug' : '--jitless',
```

Comment:
[nit] Maybe we can replace the `--no-opt` with `--no-turbofan` as it's just an alias. To keep it more consistent with the second option: `--no-sparkplug`?